content/events/2021-01-sv-with-ont/index.md

```diff
@@ -6,7 +6,7 @@ tease: "A GalaxyPro Webinar"
 continent: GL
 location: "GalaxyWorks, Online, Global"
 location_url: "https://galaxyworks.io/webinars/"
-external_url: "/events/2021-01-sv-wtih-ont/sv-with-ont.pdf"
+external_url: "/events/2021-01-sv-with-ont/sv-with-ont.pdf"
```

Comment:
Is it worth leaving a redirect to the original url? Not sure if anyone's hotlinked to it.

Reply:
Probably not for this one